MPU/src/mpuCanHandlers.cpp

```diff
@@ -55,33 +56,52 @@ void canHandler_CANMSG_MOTORMOTION(const CAN_message_t &msg)
 {
     //angular motor speed is found at bytes 2 and 3
     int16_t motorSpeed = ((msg.buf[3] << 8) | msg.buf[2]) / 10;
-    Serial.print("Motorspeed: ");
-    Serial.println(motorSpeed);
-    mpu.setMotorSpeed(motorSpeed);
+    motorController.setMotorSpeed(motorSpeed);
 }
 
 void canHandler_CANMSG_BMSCHARGINGSTATE(const CAN_message_t &msg)
 {
-    mpu.enableBMSChargingMode();
+    bms.enableChargingMode();
 }
 
 void canHandler_CANMSG_MOTORETEMP3(const CAN_message_t &msg)
 {
     int16_t motorTemp = (msg.buf[5] << 8) | msg.buf[4];
-    Serial.print("MotorTemp: ");
-    Serial.println(motorTemp / 10);
-    mpu.setMotorTemp(motorTemp);
+    motorController.setRadiatorTemp(motorTemp);
 }
 
 void canHandler_CANMSG_BMSCURRENTS(const CAN_message_t &msg)
 {
-    mpu.CANLineVerified();
+    CANLineVerified();
+
     uint16_t dischargeCurrentLimit = (msg.buf[0] << 8) | msg.buf[1];
-    mpu.setCurrentLimit(dischargeCurrentLimit);
+    bms.setCurrentLimit(dischargeCurrentLimit);
+
     uint16_t chargeCurrentLimit = (msg.buf[2] << 8) | msg.buf[3];
-    mpu.setChargeCurrentLimit(chargeCurrentLimit);
+    bms.setChargeCurrentLimit(chargeCurrentLimit);
+
+
     int16_t currentDraw = (msg.buf[4] << 8) | msg.buf[5];
-    mpu.bmsCurrentProcess(currentDraw);
+    bms.setCurrentDraw(currentDraw);
+    
+    if(!boosting_debounce.isTimerExpired())
```

Comment:
Uhhhh how did the boosting stuff get here?? This wasn't in CanHandlers before...

Reply:
And I highly doubt it should be now since we dont even use boost mode anymore

Reply:
Are we completely removing boost from the car's code entirely?

Reply:
Yee